From 7495f3db9490f9a1c40603ae84e9c5341b2854d2 Mon Sep 17 00:00:00 2001
From: Otto Modinos <ottomodinos@gmail.com>
Date: Tue, 28 May 2024 13:38:42 +0300
Subject: [PATCH 2/2] fix for `cmocka` lacking `LargestIntegralType`

In Void Linux, we've patched our `cmocka` headers to replace all the
`LargestIntegralType` definitions and uses with the standard C `uintmax_t`.
---
 source4/auth/tests/sam.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/source4/auth/tests/sam.c b/source4/auth/tests/sam.c
index a2bc4a5665e..6e8069c508d 100644
--- a/source4/auth/tests/sam.c
+++ b/source4/auth/tests/sam.c
@@ -82,8 +82,8 @@ int __wrap_samdb_msg_add_int64(
  * Mock implementations
  *****************************************************************************/
 
-static int check_dn(const LargestIntegralType left_value,
-		    const LargestIntegralType right_value)
+static int check_dn(const uintmax_t left_value,
+		    const uintmax_t right_value)
 {
 	/*
 	 * We have to cast away const so we can get the linearized form with
-- 
2.45.1

